src/appMain/routes/settings/index.js

```diff
@@ -108,8 +108,9 @@ class Settings extends React.Component {
 
   handleSaveValidity = (settings) => {
     const orgId = this.props?.detail?.metadata.id;
+    const accId = this.props.userAndRoleDetail.metadata.id;
```

Comment:
why are we passing in acc id ?

Reply:
So that if settings are made w.r.t to accounts as well, this can be used. But since now, we are dealing with org settings we can get rid of this.

Reply:
reverted